AppDevAnnouncements/Classes/AnnouncementNetworking.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  AnnouncementNetworking.swift
+//  AppDevAnnouncements
+//
+//  Created by Gonzalo Gonzalez on 11/17/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+import FutureNova
+
+public class AnnouncementNetworking {
```

Comment:
when is this going to be created?

Reply:
take a look under the Test Coverage. this can be created whenever, it's when the retrieveAnnouncement call is made that matters